### ISSUE TITLE
Documentatie: toegankelijkheid uitleggen van 2px dotted/dashed black+white focus outline

### ISSUE THREAD

**Issue by Robbert** [nl-design-system/utrecht#1996]
We hebben [documentatie over focus design](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/common-patterns-focus--docs)

- [ ] controleren of aanpak van Utrecht WCAG 2.2 proof is
- [ ] uitleggen hoe de design tokens gekozen moeten worden om WCAG 2.2 proof te zijn
- [ ] uitleggen waarom default browser outline wel of niet een goed idee is (niet een goed idee lijkt me, je kunt geen contrast garanderen met achtergrondkleur als de focus kleur onbekend is)

**Comment by sergei-maertens** [comment]
wat betreft default browser behaviour - onlangs realiseerden we dat firefox op ubuntu de ubuntu theme kleur als outline color gebruikt. bij de ene persoon was de focus ring dus oranje en bij de andere blauw. blauw gaf inderdaad onvoldoende contrast tegen blauwe buttons, dus een goed idee om niet de default browser styles te gebruiken
